### ISSUE TITLE
Acknowledge removal of job with removeRepeatable

### ISSUE THREAD

**Issue by kraten** [taskforcesh/bullmq#1235]
I found two issues while using the queue removeRepeatable method:
1. It should throw some error when repeatable job doesn't exist with the provided arguments. Instead it just returns a promise that resolves to null value.
2. Even when the repeatable job options are correct and job is removed, the promise gets resolved to null. It would be better to have some return value to know that the job got removed.

To summarize, I couldn't find a way to distinguish if the repeatable job is actually removed or the options are incorrect, as they both resolve to null value.

**Comment by roggervalf** [comment]
hi @kraten, this is related to this issue https://github.com/taskforcesh/bullmq/issues/1211, for now we are only emiting the removed event, I marked this one as an enhancement

**Comment by manast** [comment]
Fixed here https://github.com/taskforcesh/bullmq/pull/1239
